tests/unit/downloader/test_downloader.py

```diff
@@ -0,0 +1,336 @@
+import json
+from unittest import TestCase
+
+from ingest.downloader.downloader import XlsDownloader
+
+
+class XlsDownloaderTest(TestCase):
+    def test_convert_json__has_no_modules(self):
+        # given
+        entity_list = [{
```

Comment:
I would remove the data definition from the test. It would make it more readable.

Reply:
You can always collapse it. This way everything in one place.

Reply:
First of all, I am not always with the IDE that can collapse stuff.
Second, this is too much details for reading the tests.
Third, when you start doing iot for several tests, patterns emerge, and you can reuse test code.